sass/components/_channel-header__icon.scss

```diff
@@ -61,6 +61,16 @@
     padding: 0 6px;
 }
 
+.channel-header__icon--left {
+    padding: 4px;
+    margin: 0 4px 0 0;
+    height: unset;
+    .icon {
+        width: 16px;
+        height: 16px;
```

Comment:
Icons are displaying much larger than designed.
Leave the `.icon` selector as you have it, but change the `svg` selector to use:
```
width:12px;
height: 12px;
```